static/scripts/administration.js

```diff
@@ -298,9 +298,7 @@ $(document).ready(() => {
 				printQRs(
 					users.map((user) => ({
 						href: user.registrationLink.shortLink,
-						title:
-							user.fullName
-							|| `${user.firstName} ${user.lastName}`,
+						title: `${user.firstName} ${user.lastName}`,
```

Comment:
why did you removed user.fullName, it will pick user.firstName user.lastName if user.fullName is empty

Reply:
yes true, but I think there is no more such thing like fullName